loopchain/app/ui/console.py

```diff
@@ -0,0 +1,52 @@
+import asyncio
+import IPython
```

Comment:
Is `IPtyhon` library essential on running loopchain?
What is the purpose of the `ui.console`?

Reply:
The console is a part of LFT application. It is for easy debugging and operate node's action in runtime.
It is like web3.js or web3.py in ethereum.

It has to be installed in develop mode.
But now we will use LFT's console itself.